packages/tools/src/tools/annotation/ArrowAnnotateTool.ts

```diff
@@ -405,11 +405,11 @@ class ArrowAnnotateTool extends AnnotationTool {
 
   touchTapCallback = (evt: EventTypes.TouchTapEventType) => {
     if (evt.detail.taps == 2) {
-      this.doubleClickCallback(evt as unknown as EventTypes.MouseUpEventType);
+      this.doubleClickCallback(evt as EventTypes.TouchTapEventType);
```

Comment:
should just be `evt`

Reply:
should be evt since `touchTapCallback = (evt: EventTypes.TouchTapEventType) => {`